client/vat/form/tasks/vat-number-task.tsx

```diff
@@ -41,6 +45,11 @@ const getVatPrefix = () => {
 
 const getVatTaxIDName = () => {
 	switch ( wcpaySettings.accountStatus.country ) {
+		case 'AU':
+			// Note – AU GST numbers are actually ABN or ACN numbers.
+			// https://vatstack.com/articles/australian-business-number-abn-validation
+			// https://business.gov.au/registrations/register-for-taxes/tax-registration-for-your-business
+			return __( 'GST Number', 'woocommerce-payments' );
```

Comment:
I'm somewhat familiar with this AU, and it seems a bit abnormal to call it a "GST number". Normally, this would instead be labelled as "ABN".

Reply:
Good point, thanks for catching that :) in NZ it's the opposite, your GST is your IRD number, but you label it as `GST number` when appropriate.

Reply:
Actually wait – no, my comment explains my reasoning. 

The value in this field will be an ABN OR an ACN, depending on the merchant. So we either need to be smarter about which is correct, or have something generic (`GST number`) and a hint that it would your ABN or ACN.

Reply:
But … if AU companies are used to putting an ACN in an ABN field (which could be the case) … then `ABN` would work fine too 🤷🏻

Reply:
> AU companies are used to putting an ACN in an ABN field

AFAIK an `ACN` is not used for GST-related things. A company with an `ACN` needs to have an `ABN` _as well_, which they'll provide to us in our field.

We'll only want the [merchant's `ABN`](https://www.ato.gov.au/businesses-and-organisations/gst-excise-and-indirect-taxes/gst/registering-for-gst), as this is what we [require to provide on the invoice](https://www.ato.gov.au/businesses-and-organisations/gst-excise-and-indirect-taxes/gst/tax-invoices).

[Example: Bunnings](https://connectonline.asic.gov.au/RegistrySearch/faces/landing/panelSearch.jspx?searchType=OrgAndBusNm&searchText=008672179)
* ACN: 008 672 179
* ABN: 26 008 672 179

Reply:
In the URL you shared in the code comments https://vatstack.com/articles/australian-business-number-abn-validation, it looks like they're getting the `ABN` via the `ACN` using the [ABN lookup tool](https://abr.business.gov.au/).

We can do this too, but it is added complexity, when it would be easier to require an `ABN` from the merchant rather than have to determine the merchant's `ABN` based on the `ACN` they've provided.

Reply:
Aha! Thanks for digging in to that and clarifying!

Reply:
Fixed up here f1904a8495d58b0c9165c04da01d0814c1c85c9d – but feel free to jump on the PR / suggest more.